GameServerLib/Logic/Players/PlayerManager.cs

```diff
@@ -28,7 +30,7 @@ public void AddPlayer(KeyValuePair<string, PlayerConfig> p)
             };
             var player = new ClientInfo(
                 p.Value.Rank,
-                ((p.Value.Team.ToLower() == "blue") ? TeamId.TEAM_BLUE : TeamId.TEAM_PURPLE),
+                p.Value.Team.ToLower() == "blue" ? TeamId.TEAM_BLUE : TeamId.TEAM_PURPLE,
```

Comment:
I think this should be extracted to a function, rather than parsing here using an arbitrary magic string value for identification.